frontend/src/plugin-extensions/demo-plugin/http-server.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -u
+
+PUBLIC_PATH="$1"
+shift
+SERVER_OPTS="$@"
+
+http-server $PUBLIC_PATH -p 9001 -S true -C /var/serving-cert/tls.crt -K /var/serving-cert/tls.key -c-1 --cors $SERVER_OPTS
```

Comment:
Pass certs from dockerfile. So that it works locally without having to supply these cert files.